packages/web/components/chart/price-historical-v2.tsx

```diff
@@ -58,3 +64,41 @@ export const HistoricalPriceChartV2: FunctionComponent<{
     />
   );
 });
+
+export const HistoricalPriceChartHeaderV2: FunctionComponent<{
```

Comment:
Would be good to eventually replace v1 with v2 for cleanup

Reply:
Ye I'll do it, right now the old header is still used for the CL pools, so maybe I'll do it after the info page PR